tutel/custom/custom_kernel.cpp

```diff
@@ -444,6 +460,54 @@ static void nccl_all_to_all_gather_async(
   g_cuda_events[0].record(get_nccl_stream());
 }
 
+static void all_to_all(torch::Tensor &output, torch::Tensor &input, const char *algo) {
```

Comment:
This should be all_to_all_async because reaching the end of this function doesn't promise everything in a2a is done.

Reply:
BTW, I think you should leverage something like CustomStreamRelease and CustomStreamAcquire to ensure proper order between computation and a2a, else computation might start when a2a is not completed, because they're on different streams.

Reply:
updated name
change to default stream so that computation and a2a are in same stream in this function